cuqi/distribution/_distribution.py

```diff
@@ -383,3 +384,59 @@ def __repr__(self) -> str:
             return "CUQI {}. Conditioning variables {}.".format(self.__class__.__name__,self.get_conditioning_variables())
         else:
             return "CUQI {}.".format(self.__class__.__name__)
+        
+    # The following methods define algebraic operations on distributions
+    # The return type is a RandomVariable instance
+        
+    def as_random_variable(self):
+        """Return a RandomVariable instance of this distribution"""
+        return RandomVariable(self)
+    
+    @property
+    def rv(self):
+        """Return a RandomVariable instance of this distribution"""
+        return self.as_random_variable()
+    
+    def __add__(self, other) -> RandomVariable:
```

Comment:
Re the first point I liked to keep the code as is to avoid any potential issues with the order of operations etc. Here it is clear that the underlying method is called with the same arguments.

Regarding the 2nd point I created an issue  #259

Reply:
Makes sense 👍